Extensions/TileMap/JsExtension.js

```diff
@@ -675,14 +679,22 @@ module.exports = {
      * Return the width of the instance, when it's not resized.
      */
     RenderedTileMapInstance.prototype.getDefaultWidth = function () {
-      return this._pixiTileMapData?.tilemapWidth / this._pixiObject.scale.x;
+      return (
+        (this._pixiTileMapData
+          ? this._pixiTileMapData.tilemapWidth
+          : this._pixiObject.width) / this._pixiObject.scale.x
```

Comment:
I think this is overly complex. If we don't have a pixiTileMapData, then we don't have any info at all about the tilemap, so we don't know the size. So we return 0:

```
if (! this._pixiTileMapData) return 0;

return this._pixiTileMapData.tilemapWidth / this._pixiObject.scale.x;
```

We never use `this._pixiObject.width`, because the object might not be of the proper size because of the tiles not being set (the "empty tiles").

Reply:
what if the object was resized?